auditapi/README.md

```diff
@@ -49,15 +47,15 @@ Default configuration comes with `Servers` field that contains server objects as
 For using other server than the one defined on index 0 set context value `sw.ContextServerIndex` of type `int`.
 
 ```golang
-ctx := context.WithValue(context.Background(), sw.ContextServerIndex, 1)
+ctx := context.WithValue(context.Background(), auditapi.ContextServerIndex, 1)
 ```
 
 ### Templated Server URL
 
 Templated server URL is formatted using default variables from configuration or from context value `sw.ContextServerVariables` of type `map[string]string`.
 
 ```golang
-ctx := context.WithValue(context.Background(), sw.ContextServerVariables, map[string]string{
+ctx := context.WithValue(context.Background(), auditapi.ContextServerVariables, map[string]string{
```

Comment:
As per issue linked to this PR, since we are already modifying this Readme file mustache template, can we add instructions about auth token generation in this readme? Similar to what we have in the other SDKs.
When it comes to the token generation in GO, you can presently find this documented the [root README file](https://github.com/phoenixnap/go-sdk-bmc/blob/master/README.md)

Reply:
Updated the READMEs with an OAuth example.